exercises/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,20 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+function myGreeting (greet) {
+    return greet;
+}
+
+var result = myGreeting ('HELLO') + ' ' + mentor1.toUpperCase();
```

Comment:
You should have put these repeated statements in a method, because the method actually does nothing

Reply:
Hi thank you for your review I will go back and make the necessary changes